README.md

```diff
@@ -86,6 +86,66 @@ To enable verbose debug mode use:
 ./build/src/edgesec -c ./build/dev-config.ini -ddddd
 ```
 
+## Docker for cross compile
+
+The following commands also work using the `podman` container runtime (recommended on Linux):
+
+```console
+alias docker=podman
+```
+
+### Configure
+
+#### Step 1
+
+In `config.mak` change the `TARGET` key to the desired architecture (see details [https://github.com/richfelker/musl-cross-make](https://github.com/richfelker/musl-cross-make)):
```

Comment:
Tbh, I'd rather there be a `--build-arg TARGET=...` in the `docker build` command, (see [`docker build --build-arg`](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg)) but that can be another PR/issue.

Reply:
Yes, lets make another PR/issue.